### PR TITLE
Add gender field to student

### PR DESCRIPTION
**Pull request by yhelen**

### CODE REVIEW

File: app/models/student.rb [hunk 1]
```diff
@@ -32,10 +32,16 @@ class Student < ApplicationRecord
   validates :osis, length: { is: 9 },
                    numericality: { greater_than: 0,
                                    message: 'cannot be negative' }
+  validates :gender, inclusion: { in: %w(Female Male Other) }
```
Comment (justinkim) [inline app/models/student.rb]: Not that it matters, but it doesn't have to be capitalized as a value in the db, but possibly in presentation.

So allowed values could just be `%w(female male other)`. Side effect would be it's easy to compare with `String.downcase`.

File: db/migrate/20180223144210_add_gender_to_students.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddGenderToStudents < ActiveRecord::Migration[5.1]
+  def change
+    add_column :students, :gender, :string
```
Comment (justinkim) [inline db/migrate/20180223144210_add_gender_to_students.rb]: Index please

File: app/models/student.rb (hunk 1 above)
Comment (yhelen) [reply app/models/student.rb]: can i just keep it as such

Comment (justinkim) [reply app/models/student.rb]: ya

**Pull request merged**